doc/ref_model/chapters/chapter03.md

```diff
@@ -7,9 +7,8 @@
 * [3.2 Virtual Resources](#3.2)
   * [3.2.1 Tenant](#3.2.1)
   * [3.2.2 Compute](#3.2.2)
-  * [3.2.3 Storage](#3.2.3)
-  * [3.2.4 Network](#3.2.4)
-  * [3.2.5 Availability Zone](#3.2.5)
+  * [3.2.3 Storage](#3.2.4)
```

Comment:
"[3.2.3 Storage](#3.2.4)" should be "[3.2.3 Storage](#3.2.3 )"

"[3.2.4 Availability Zone](#3.2.5)" should be "[3.2.4 Availability Zone](#3.2.4  )"  (the extra space as Github keeps changing to an Issue -- but please delete)

Reply:
@pgoyal01 resolved in https://github.com/cntt-n/CNTT/pull/1536/commits/e3e5ca93fc3a528f62a84d60de15d024af6373c9

Reply:
@rabi-abdel Thanks. 3.2.3 Storage still references 3.2.4 instead of 3.2.3

Reply:
I committed a change to the anchor.